tests/matrix_free/constraint_info_01.output

```diff
@@ -5,7 +5,7 @@ DEAL::(0,0) (1,3) (3,5) (5,7) (7,9) (8,12) (9,15) (10,18)
 DEAL::
 DEAL::11 11 2 11 2 11 2 2 2 2 
 DEAL::(0,0) (0,0) (0,0) (0,0) (0,0) (0,0) (1,0) (1,0) (0,0) (0,0) (0,0) (1,0) (0,0) (1,0) (0,0) (1,0) (0,0) (0,0) 
-DEAL::(0,0) (1,3) (3,5) (5,7) (7,9) (8,12) (9,15) (10,18) (10,18) 
+DEAL::(0,0) (1,3) (3,5) (5,7) (7,9) (8,12) (9,15) (10,18) 
```

Comment:
I remember I was confused about this output when I wrote it. Good to know what the reason in :+1: